app/views/sequences/show.html.haml

```diff
@@ -42,16 +42,18 @@
   .activities
     - @sequence.lightweight_activities_sequences.each do |las|
       - activity = las.lightweight_activity
+      - activity_path_params = @sequence_run ? [:run_key => @sequence_run.run_for_activity(activity).key] : []
+      - activity_path_link = pass_white_list_params(sequence_activity_path(@sequence, activity, *activity_path_params))
```

Comment:
Is this passing the run_key as a URL param instead of adding to the path of the URL?  We'd want it to be part of the URL path for consistency.

Reply:
I was worried that updating the route would break any bookmarked urls since they would not have the `/:run_key` suffix in the path.  If you think that isn't a problem I can update the code.

Reply:
The URL without the run_key in the path should still work, but it should redirect to a URL with the run_key in path.